### PR TITLE
Add initial documentation for the historical data import.

### LINKED ISSUE #3370: Documentation: Historical Data Import
This issue is to begin user documentation for docs.woocommerce.com. For the sake of iterating, and being able to tie this work out to the repository, lets craft these in markdown, and place them inside the /docs/woocommerce.com folder.

For the Historical Data import lets cover a bit about how the process works ( batching, action scheduler ), how selecting Skip previously imported customers works, and when/why it might be good to use Delete Previously Imported Data.

I think it would also be a good place to link to Action Scheduler documentation on running job processors via wp cli

### PR DESCRIPTION
**Pull request by jeffstieler**
Fixes #3370.

Adds documentation about the historical data import.

### Detailed test instructions:

- Check the formatting and read the doc
- Check for accuracy

### CODE REVIEW

File: docs/woocommerce.com/analytics-historical-data-import.md [hunk 1]
```diff
@@ -1 +1,52 @@
...
+
+#### Initial Data Import
+
+The historical data import uses [Action Scheduler](https://actionscheduler.org/) to queue jobs that process the existing data on your store, building tables of aggregate data to be used in Reports.
+
+Before version 3.0.0, Action Scheduler [used custom post types](https://actionscheduler.org/faq/) to manage queue data. To avoid problems queuing large numbers of items, the import jobs are queued and processed in batches to reduce the likelihood of execution time limits on servers/hosts with limited resources.
+
+#### Subsequent Data Updates
+
+All Customers and Orders that get created after WooCommerce 3.9 (or the WooCommerce Admin plugin) is installed are automatically queued for processing. Only historical data requires manual intevention to process.
```
Comment (rrennick) [inline docs/woocommerce.com/analytics-historical-data-import.md]: > that get created after WooCommerce 3.9 (or the WooCommerce Admin plugin)

That sort of reads as though you don't need WC Admin if you have WC 3.9.

File: docs/woocommerce.com/analytics-historical-data-import.md [hunk 2]
```diff
@@ -1 +1,52 @@
-# Importing Historical Data
\ No newline at end of file
+# Importing Historical Data
+
+When you install WooCommerce 3.9 (or the WooCommerce Admin plugin) the new Analytics functionality needs to process the historical data in order to populate new Report pages. This is not done automatically - extremely large or high volume stores might want to schedule a maintenance window in which to import their historical data.
```
Comment (rrennick) [inline docs/woocommerce.com/analytics-historical-data-import.md]: Same reading as below.

File: docs/woocommerce.com/analytics-historical-data-import.md (hunk 1 above)
Comment (jeffstieler) [reply docs/woocommerce.com/analytics-historical-data-import.md]: I should have noted - this is meant to read whatever version of core WC we merge into.

Comment (rrennick) [reply docs/woocommerce.com/analytics-historical-data-import.md]: Gotcha. How about we put version 4.0 then if need be change it later?

**Pull request merged**